core/primitives/src/testing.rs

```diff
@@ -36,7 +36,7 @@ pub struct KeyStore {
 #[cfg(feature = "std")]
 impl KeyStore {
 	/// Creates a new instance of `Self`.
-	pub fn new() -> crate::traits::BareCryptoStorePtr {
+	pub fn new_key_store_ptr() -> crate::traits::BareCryptoStorePtr {
```

Comment:
might be better before

Reply:
I am not sure - `new` usually implies returning `Self` so it might be confusing. Also, the comment is a bit misleading.

Reply:
This is why I changed it here, see https://rust-lang.github.io/rust-clippy/master/#new_ret_no_self.
I can change the comment as well.